### PR TITLE
chore(BUX-601): update keygen names

### PR DESCRIPTION
**Pull request by wregulski**
### TODO:
[x] Update package names for spv-wallet-go-client (previously go-buxclient).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ FROM --platform=$TARGETPLATFORM bitnami/kubectl:latest AS final
 
 WORKDIR /src
 
-ENV SECRET_NAME=bux-keys
+ENV SECRET_NAME=bsv-keys
```
Comment (Nazarii-4chain) [inline Dockerfile]: Shouldn't these be `spv-keys` ?

File: README.DOCKER.template.md [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
 `SECRET_NAME`: The name of the secret.
 `XPUB_KEY_NAME`: The name of the key storing the public key.
 `XPRV_KEY_NAME`: The name of the key storing the private key.
 
 ### Environment Variables
+
 The Docker image accepts the following environment variables:
 
-`SECRET_NAME`: (Default: _bux-keys_) The name of the Kubernetes secret.
+`SECRET_NAME`: (Default: _bsv-keys_) The name of the Kubernetes secret.
```
Comment (Nazarii-4chain) [inline README.DOCKER.template.md]: Same here

File: Dockerfile (hunk 1 above)
Comment (dorzepowski) [reply Dockerfile]: rather `spv-wallet-keys`

File: README.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
-# Bux Key Generator
-This tool generates public and private key pairs for BuxOrg applications. The generated keys are stored in separate files (xpub_key.txt and xprv_key.txt).
+# SPV Wallet Admin Keygen
+
+This tool generates public and private key pairs for bitcoin-sv (SPV-Wallet) applications. The generated keys are stored in separate files (xpub_key.txt and xprv_key.txt).
```
Comment (dorzepowski) [inline README.md:3]:
```suggestion
This tool generates public and private key pairs for SPV Wallet applications. The generated keys are stored in separate files (xpub_key.txt and xprv_key.txt).
```

**Review by dorzepowski — CHANGES_REQUESTED** [review]

File: README.DOCKER.template.md (hunk 2 above)
Comment (wregulski) [reply README.DOCKER.template.md]: I renamed it to `spv-wallet-keys` if you don't mind

File: README.md (hunk 3 above)
Comment (wregulski) [reply README.md:3]: done

File: Dockerfile (hunk 1 above)
Comment (wregulski) [reply Dockerfile]: `spv-wallet-keys` I updated it with this name.

File: go.mod [hunk 4]
```diff
@@ -1,17 +1,12 @@
-module BuxOrg/bux-admin-key-generator
+module github.com/bitcoin-sv/spv-wallet-admin-keygen
 
 go 1.21
 
 toolchain go1.21.4
 
+require github.com/BuxOrg/go-buxclient v0.4.0
```
Comment (Nazarii-4chain) [inline go.mod]: We will rename this after go-buxclient renaming ?

**Review by Nazarii-4chain — APPROVED** [review]

**Review by dorzepowski — APPROVED** [review]

File: go.mod (hunk 4 above)
Comment (wregulski) [reply go.mod]: image

Yes, exactly

**Comment by Nazarii-4chain** [comment]
Testing

**Comment by Nazarii-4chain** [comment]
Tested

**Pull request merged**